Orochi/GpuMemory.h

```diff
@@ -91,8 +91,27 @@ class GpuMemory final
 		*this = std::move( tmp );
 	}
 
+	void resizeAsync( const size_t new_size, const bool copy = false, oroStream stream = 0 ) noexcept
```

Comment:
Could you please also add the doxy notes similar to the `resize` API ? 
I think it's very important to indicate that the copy might not be finished even if the function call returns.

Reply:
Alternatively, you could choose to merge this together with the original `resize` function by providing an additional `bool async = false` parameter.